### DISCUSSION TITLE
How to switch lighttpd to Nginx or Apache2 ?

### DISCUSSION THREAD

**Discussion by kevinkchoi** [RaspAP/raspap-webgui#908] [Q&A]
Hi all,
RaspAP is a great program. I love it.

I'm currently running RACHEL server which runs Kolibri, Kiwix, etc. on a Nginx web service.
if I install RaspAP, it installs lighttpd which conflicts with Nginx.
So, I disabled Lighttpd and came back to Nginx. Unfortunately, on Nginx (and even on Apache2), the menu click of RaspAP home screen does not provide anything and creates 404 error.

Can someone help me to resolve this problem, please?
I need to keep nginx for other server softwares as it took long time to tune them in...

Thank you for your kind help in advance.
Kevin

**Comment by billz** [comment]
Here you go https://docs.raspap.com/faq/#webport

**Reply by Shumaher** [reply]
So, is there a way to host RaspAP via nginx or Apache2?
I already have nginx on my RPi, no really need to have second web-server installed...

**Reply by billz** [reply]
Yes, it's entirely possible to use nginx or Apache. For the former, you only need to configure a reverse proxy that forwards PHP requests to an interpreter (usually PHP-FPM) using FastCGI.
